images/installer/Dockerfile

```diff
@@ -22,7 +22,7 @@ USER root
 # configurations for the two images.
 RUN mkdir -p /usr/share/ansible/ && ln -s /opt/app-root/src /usr/share/ansible/openshift-ansible
 
-RUN INSTALL_PKGS="skopeo" && \
+RUN INSTALL_PKGS="skopeo openssl java-1.8.0-openjdk-headless python-passlib httpd-tools" && \
```

Comment:
@juanvallejo  Hmm, not related to this but if we depend on skopeo we should add that to the rpm spec file. Is that a control host requirement or target host?